.github/pr_management/ready_for_review_comment/main.ts

```diff
@@ -0,0 +1,64 @@
+import * as core from '@actions/core'
+import * as github from '@actions/github';
+import { postComment, validateChecksOnPrHead, dropOngoingLabel, addAppropriateReviewLabel, reviewKeywords } from "../common"
+
+// params to set for api requests
+// check https://github.com/actions/toolkit/blob/main/packages/github/src/context.ts to figure out what's being responded
+const issue_number = github.context.issue.number;
+
+/**
+ * This is the main function of this file
+ */
+async function run() {
+    try {
+        // all comments trigger this workflow
+        const doesCommentContainKeywords = filterCommentBody();
+        if (!doesCommentContainKeywords) return;
+
+        const valid : boolean = await validate();
+        if (!valid) return;
+
+        await dropOngoingLabel();
+        await addAppropriateReviewLabel();
+    } catch (ex) {
+        core.info(ex);
+        core.setFailed(ex.message);
+    }
+}
+
+// return if comment body has the exact keywords
+function filterCommentBody() : boolean {
+    const issueComment = github.context.payload.comment.body;
+    const hasKeywords = issueComment.search(reviewKeywords) !== -1;
```

Comment:
should we also check that the comment is by the author of the PR?

Reply:
This makes sense if most PRs are only worked by the author, so I've added it now